store/index.jsx

```diff
@@ -9,13 +9,13 @@ let globalStore = null
 
 useStaticRendering(isServer)
 
-export const initStore = ({ dataProvider, activity }) => {
+export const initStore = () => {
   // on the server-side a new instance is created for each page request
   // as we don't wan't to mix between users/requests, etc.
-  if (isServer) return new RootStore(dataProvider, activity)
+  if (isServer) return new RootStore()
 
   if (!globalStore) {
-    globalStore = new RootStore(dataProvider, activity)
+    globalStore = new RootStore()
```

Comment:
Do not you want to have some initial configuration? As a user you can go to or refresh `/data-providers/86/deposit-dates` and the application should load 86th data provider if the user has an access to it.

To extend, if user is not authorised, application should redirect to login form and after login it should redirect back to 86th data provider's deposit dates but we would not do it for now.

Reply:
There is no need to initial configuration. These values are set here `https://github.com/oacore/dashboard/blob/login/pages/_app/index.jsx#L65`.


> To extend, if user is not authorised, application should redirect to login form and after login it should redirect back to 86th data provider's deposit dates but we would not do it for now.

We will need to store the URL somewhere in storage/cookie. Now the redirection is hardcoded. Let's do it in another PR.

Reply:
I would rather pass it as a `continue` parameter, cached by JS and passed again.